js/index.js

```diff
@@ -20,6 +20,7 @@ globalThis.app = createApp({
 		for (let name of this.repoNames) {
 			// this.load(name);
 			this.labels[name] = new RepoLabels(name, {syncWith: this.firstBackend});
+			this.labels[name].init();
```

Comment:
I knew you could do it!
I’m assuming if we call `this.init()` from within the constructor it will have the same issue?

Reply:
> I knew you could do it!

Thank you! 😊

> I’m assuming if we call `this.init()` from within the constructor it will have the same issue?

Yup. I also checked it to make sure it won't work. :)